solr/solr-ref-guide/src/distributed-requests.adoc

```diff
@@ -115,17 +115,19 @@ If specified, the thread pool will use a backing queue instead of a direct hando
 `fairnessPolicy`::
 Chooses the JVM specifics dealing with fair policy queuing, if enabled distributed searches will be handled in a First in First out fashion at a cost to throughput. If disabled throughput will be favored over latency. The default is `false`.
 
-`shardsWhitelist`::
+In addition, `HttpShardHandlerFactory` also depends on the following top-level property:
+
+`allowUrls`::
```

Comment:
I think you should be making reference to `format-of-solr-xml.adoc` where this setting should be canonically documented.  That said, I understand why this distributed-requests page has information on this.

Reply:
+1